Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+@testable import FlowMagic_SDK
+
+final class FlowMagicSDKTests: XCTestCase {
+    func testExample() throws {
+
+    }
+}
```

Comment:
Why add this file?  The only method it contains is a no-op.

Reply:
Also, does this mean you have no tests?  If so, add some.

Reply:
Test cases are added in a separate [PR](https://github.com/KedarDhere/FlowMagic-SDK/pull/7). This PR includes only file organization-related changes.